ui/js/selectors/file_info.js

```diff
@@ -110,16 +121,16 @@ export const selectFileInfosPublished = createSelector(
 
 export const selectFileInfosByUri = createSelector(
   selectClaimsByUri,
-  selectFileInfosByOutpoint,
-  (claimsByUri, byOutpoint) => {
+  selectFileInfosBySdHash,
+  (claimsByUri, bySdHash) => {
     const fileInfos = {};
     const uris = Object.keys(claimsByUri);
 
     uris.forEach(uri => {
       const claim = claimsByUri[uri];
       if (claim) {
-        const outpoint = `${claim.txid}:${claim.nout}`;
-        const fileInfo = byOutpoint[outpoint];
+        const sd_hash = claim.value.stream.source.source;
```

Comment:
`.source.source`? Guessing this is correct, but please tell me who to complain to :P

Reply:
The schema/lbryum/daemon guys.